apps/spruce/src/pages/version/useQueryVariables/index.ts

```diff
@@ -17,37 +19,39 @@ export const useQueryVariables = (
   const { limit, page } = usePagination();
   const queryParams = parseQueryString(search);
   const {
-    [PatchTasksQueryParams.Duration]: duration,
-    [PatchTasksQueryParams.Sorts]: sorts,
+    [TableQueryParams.Sorts]: sorts,
     [PatchTasksQueryParams.Variant]: variant,
     [PatchTasksQueryParams.TaskName]: taskName,
     [PatchTasksQueryParams.Statuses]: statuses,
     [PatchTasksQueryParams.BaseStatuses]: baseStatuses,
   } = queryParams;
 
-  // This should be reworked once the antd tables are removed.
-  // At the current state, sorts & duration will never both be defined.
   let sortsToApply: SortOrder[] = [];
-  const opts = {
-    sortByKey: "Key" as "Key",
-    sortDirKey: "Direction" as "Direction",
-    sortCategoryEnum: TaskSortCategory,
-  };
-  if (sorts) {
-    sortsToApply = parseSortString<
-      "Key",
-      "Direction",
-      TaskSortCategory,
-      SortOrder
-    >(sorts, opts);
-  }
-  if (duration) {
+  const taskSortCategories: string[] = Object.values(TaskSortCategory);
+  const parsedSortBy = getString(queryParams[TableQueryParams.SortBy]);
+  const sortBy = taskSortCategories.includes(parsedSortBy)
+    ? (parsedSortBy as TaskSortCategory)
+    : undefined;
+  const parsedSortDir = getString(queryParams[TableQueryParams.SortDir]);
+  const sortDir =
+    parsedSortDir === SortDirection.Desc
+      ? SortDirection.Desc
+      : SortDirection.Asc;
```

Comment:
I think you can type assert instead
```suggestion
  const sortDir = getString(queryParams[TableQueryParams.SortDir]) as SortDirection;
```

Reply:
I removed this code because I adjusted the table back to using `sorts` only